src/components/Footer.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Fetch from 'node-fetch';
+
+export default React.createClass({
+
+  componentDidMount() {
+    fetch('https://static.zooniverse.org/zoo-footer/zoo-footer.json')
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        this.setState(data);
+      });
+
+  },
+  getInitialState: () => {
+  return {};
+  },
+  render() {
+   let space = (this.state.projectsList && this.state.projectsList[0].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let naturalWorld = (this.state.projectsList && this.state.projectsList[1].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let humanities = (this.state.projectsList && this.state.projectsList[2].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let bioPhys = (this.state.projectsList && this.state.projectsList[3].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+    console.log(this.state);
+    return (
+      <div className="site-footer">
+
+    <section className="footer-zooniverse">
+        <div className="container">
+            <svg  preserveAspectRatio="xMidYMid" viewBox="0 0 59 55">
+                <path d="M59.010,4.828 C58.792,4.534 55.644,6.526 50.665,10.026 C45.841,3.925 38.349,0.000 29.319,0.000 C13.281,0.000 1.943,12.535 1.943,27.500 L1.943,27.652 C1.943,33.588 3.704,39.116 6.829,43.628 C2.380,47.346 -0.217,49.743 -0.006,50.029 C0.205,50.316 3.191,48.438 7.936,45.113 C12.764,51.134 20.210,55.000 29.170,55.000 C45.209,55.000 56.547,42.466 56.547,27.500 L56.547,27.348 C56.547,21.492 54.834,16.031 51.787,11.553 C56.472,7.651 59.227,5.122 59.010,4.828 ZM11.566,27.500 L11.566,27.348 C11.566,17.017 18.801,8.584 29.170,8.584 C35.000,8.584 39.886,11.297 43.048,15.510 C39.949,17.783 36.534,20.329 32.928,23.060 C31.994,22.252 30.802,21.767 29.502,21.767 C26.510,21.767 24.085,24.334 24.085,27.500 C24.085,28.225 24.217,28.916 24.449,29.554 C20.749,32.420 17.292,35.141 14.209,37.609 C12.517,34.672 11.566,31.199 11.566,27.500 ZM46.925,27.500 L46.925,27.652 C46.925,37.984 39.688,46.416 29.319,46.416 C23.559,46.416 18.722,43.764 15.559,39.637 C18.738,37.310 22.261,34.685 25.990,31.861 C26.936,32.715 28.161,33.234 29.502,33.234 C32.494,33.234 34.919,30.667 34.919,27.500 C34.919,26.732 34.774,26.001 34.515,25.332 C38.068,22.581 41.398,19.963 44.388,17.574 C46.014,20.472 46.925,23.877 46.925,27.500 Z"></path>
+            </svg>
+            <p className="footer-zooniverse">
+              {this.state.description}
+            </p>
+            <a href="https://www.zooniverse.org/" className="button">
+              {this.state.buttonText}
+          </a>
+        </div>
+    </section>
+
+    <section className="footer-projects" ng-hide="smallFooter">
```

Comment:
1. There's some old angular stuff in here, such as `ng-hide`;
2. Rather than have lots of conditionals, and default values that may change later on, you could optionally render the project list depending on the state, so here, you have: 

```
{ renderProjectList() }
```

And somewhere else, you have a method that does something like this:

```
function renderProjectList() {
  if (this.state.projectList) {
    return (<section className="footer-projects">
      ...other project list stuff...
    );
    return null;
}
```